be/test/storage/tablet_updates_test.cpp

```diff
@@ -351,6 +420,15 @@ static ssize_t read_tablet_and_compare(const TabletSharedPtr& tablet, int64_t ve
     return read_and_compare(iter, keys);
 }
 
+static ssize_t read_tablet_and_compare_schema_changed(const TabletSharedPtr& tablet, int64_t version,
```

Comment:
read_and_compare_schema_changed and read_tablet_and_compare_schema_changed can be merged.

Reply:
changed.